src/lib.rs

```diff
@@ -921,8 +967,9 @@ pub use structopt_derive::*;
 
 use std::ffi::OsString;
 
-/// Re-export of clap
+/// Re-exports
 pub use clap;
+pub use lazy_static;
```

Comment:
Maybe we can doc(hidden). Or not, as you wish.

Reply:
The question is whether we want to make the lazy_static public API or not. Let's not.